### PR TITLE
feat(downvote): remove query field of downvote counts

### PR DESCRIPTION
**Pull request by Jasmine-liang**
Temporarily set the downvote counts to 0 and remove the query field from the response.

Met an error: `[GraphQL error]: Message: Fragment "DownvoteCommentPrivate"` is never used

### CODE REVIEW

File: src/components/Comment/FooterActions/index.tsx [hunk 1]
```diff
@@ -143,9 +140,9 @@ const BaseFooterActions = ({
   return (
     <footer
       aria-label={translate({
-        zh_hant: `${comment.upvotes} 點讚、${comment.downvotes} 點踩`,
-        zh_hans: `${comment.upvotes} 点赞、${comment.downvotes} 点踩`,
-        en: `${comment.upvotes} upvotes, ${comment.downvotes} downvotes`,
+        zh_hant: `${comment.upvotes} 點讚、${0} 點踩`,
+        zh_hans: `${comment.upvotes} 点赞、${0} 点踩`,
+        en: `${comment.upvotes} upvotes, ${0} downvotes`,
```
Comment (tx0c) [inline src/components/Comment/FooterActions/index.tsx:145]: just remove all of `${comment.downvotes} downvotes` ... don't leave a `0` there

File: src/components/Comment/FooterActions/UpvoteButton/index.tsx [hunk 2]
```diff
@@ -60,7 +59,7 @@ const UpvoteButton = ({
       unvoteComment: {
         id: comment.id,
         upvotes: comment.upvotes - 1,
-        downvotes: comment.downvotes,
+        downvotes: 0,
```
Comment (tx0c) [inline src/components/Comment/FooterActions/UpvoteButton/index.tsx:62]: just comment out, or remove the whole line, don't need a `0`

Comment (Jasmine-liang) [reply src/components/Comment/FooterActions/UpvoteButton/index.tsx:62]: Tired commenting out, the `downvotes` field is required. Discussed with Robert, could leave it as 0(could not be null also). Any thoughts?

**Review by robertu7 — APPROVED** [review]
LGTM

**Comment by tx0c** [comment]
seems caused any circle broadcast unable to load

**Pull request merged**